web/forms.py

```
@@ -1736,11 +1738,14 @@ class Meta:
             "time_limit": TailwindNumberInput(
                 attrs={"min": "0", "placeholder": "Time limit in minutes (leave empty for no limit)"}
             ),
+            "passing_score": TailwindNumberInput(attrs={"min": "50"}),
             "randomize_questions": TailwindCheckboxInput(),
             "show_correct_answers": TailwindCheckboxInput(),
             "allow_anonymous": TailwindCheckboxInput(),
+            "AI_auto_correction": TailwindCheckboxInput(),
             "max_attempts": TailwindNumberInput(attrs={"min": "0", "placeholder": "0 for unlimited attempts"}),
         }
+        exclude = ["subject"]
```

Ruff (0.8.2)

Do not use `exclude` with `ModelForm`, use `fields` instead

(DJ006)